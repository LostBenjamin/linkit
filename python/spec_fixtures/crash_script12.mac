[render]
  refresh-display

[render]
  refresh-display

