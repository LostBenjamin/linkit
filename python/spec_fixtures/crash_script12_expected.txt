Macro 'render' cannot be reassigned.

